hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/assertions/EventualRecordStreamAssertion.java

```diff
@@ -59,6 +61,17 @@ public EventualRecordStreamAssertion(final Function<HapiSpec, RecordStreamAssert
         this.assertionFactory = assertionFactory;
     }
 
+    private EventualRecordStreamAssertion(
```

Comment:
This whole area of `EventualAssertion` and `RecordStreamAssertion` is new to me ... interesting to see it.